CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Developing XTF
+
+Compile from source:
+
+```mvn clean install```
+
+# XTF git development cycle <a name="devcycle"></a>
+We use the [git flow](https://guides.github.com/introduction/flow/) workflow.
+The steps are
+
+0. Read the [guide](https://guides.github.com/).
+1. Create an account [GitHub](https://github.com/).
+2. Fork the [XTF](https://github.com/xtf-cz/xtf/) repository.
+3. Clone your fork, add the upstream XTF repository as a remote, and check out locally:
+
+        git clone git@github.com:USERNAME/xtf.git
+        cd xtf
+        git remote add upstream git@github.com:xtf-cz/xtf.git
+        git branch master
+        git checkout master
+        git pull --rebase upstream master
+
+   The steps until here only need to be executed once, with the exception being the last command: rebasing against upstream master branch
```

Comment:
Missing a dot in the end of the sentence.

Reply:
fixed